videos/videos_model.js

```diff
@@ -19,8 +19,11 @@ async function add(data) {
 }
 
 function find9() {
-  const res = db.raw("select * from videos ORDER BY random() limit 9 ");
-  return res;
+  return db("videos")
+    .select()
+    .where("video_status", "successful")
+    .orderByRaw("RANDOM()")
+    .limit(9);
 }
 
 function find() {
```

Comment:
Nice work. 
The change to video_status is inline with the update to our videos table on this pr and will importantly prevent unfinished or failed videos form returning. The switch to knex methods from raw sql does offer a more uniform feel to the file.